users/tests/test_models.py

```diff
@@ -60,12 +60,12 @@ def test_create_usergroup(self):
         group = UserGroup.objects.create(
             name='PLUG',
             location='London, UK',
-            url='http://meetup.com/plug',
+            url='https://meetup.com/plug',
```

Comment:
This is just a test, no need to change.

Reply:
I agree, I however though it would be good to have some https in tests just in case it break something. But it should better be separated tests in this case so I revert this one OK.